src/Pages/RegisterForm/ModalWindow/ModalWindow.tsx

```diff
@@ -1,11 +1,50 @@
-
-
-const ModalWindow = () => {
-  return (
-    <div>
-      
-    </div>
-  )
+import React from 'react';
+import {useNavigate} from 'react-router-dom';
+import { Button, Modal } from 'antd';
+import smile from '@/images/Homepage/smile.png';
+import './style.scss';
+interface ModalWindowProps{
```

Comment:
you have a script for prettifying the code - i would suggest to run it before making a new commit . (`npm run format`).

----


Also, there is possibility to make `vscode` to autoformat your code on file save. Not sure if these are all steps but you could try at some point if you want to: 

1. -  install extension on your `vscode` - `esbenp.prettier-vscode`, 
2. -  then go to ` vscode settings`; find   editor.formatOnSave and make sure it is checked ; 
 <img width="980" alt="image" src="https://github.com/user-attachments/assets/8db1979a-e51b-4987-939c-e4190a9c9413">

Reply:
Done